src/components/JsonLink/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import classnames from 'classnames';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faRobot } from '@fortawesome/free-solid-svg-icons';
+import styles from './index.module.scss';
+
+interface Props {
+  fileTitle: string;
+  version: string;
+}
+
+const baseURL = `https://nodejs.org/docs/latest-`;
+
+const JsonLink = ({ fileTitle, version }: Props): JSX.Element | null => {
```

Comment:
instead of fileTitle, `fileName`. Yes, it is the `title` field 😅 but it is actually the name of the file.